code/controllers/configuration/entries/general.dm

```diff
@@ -458,18 +458,14 @@
 	min_val = 0
 	max_val = 1
 
-/datum/config_entry/number/ipintel_save_good
-	default = 12
-	integer = FALSE
-	min_val = 0
+/datum/config_entry/flag/ipintel_reject_rate_limited
+	default = TRUE
 
-/datum/config_entry/number/ipintel_save_bad
-	default = 1
-	integer = FALSE
-	min_val = 0
+/datum/config_entry/flag/ipintel_reject_bad
+	default = TRUE
 
-/datum/config_entry/string/ipintel_domain
-	default = "check.getipintel.net"
```

Comment:
This is still needed because if you pay for more you get a new domain you must used